app/components/satis/dropdown/component.rb

```diff
@@ -21,7 +21,12 @@ def initialize(form:, attribute:, **options, &block)
 
         options[:input_html] ||= {}
 
-        options[:input_html][:value] = hidden_value
+        @hidden_value = hidden_value
```

Comment:
So we have options[:input_html][:value] and value where value needs to be the value without the .id?
Why can't we keep the options[:input_html][:value]?